applications/teletype/priv/templates/cnam_request.html

```diff
@@ -67,7 +67,7 @@ <h3 style="margin:0;padding:0;font-family:'Open Sans',sans-serif;color:#555555;f
                             </tr>
                             <tr>
                                 <td style="padding:10px;background-color:#f5f5f5;text-align:right;font-family:'Open Sans',sans-serif;color:#555555;font-size:13px;font-weight:400;line-height:20px;border-size:3px;border-color:#ffffff;border-right-style:solid;border-bottom-style:solid;" width="30%"><b>Number State</b></td>
-                                <td style="padding:10px;background-color:#eaeaea;font-size:13px;font-family:'Open Sans',sans-serif;color:#555555;line-height:20px;border-size:3px;border-color:#ffffff;border-bottom-style:solid;" width="70%">{{request.state}}</td>
+                                <td style="padding:10px;background-color:#eaeaea;font-size:13px;font-family:'Open Sans',sans-serif;color:#555555;line-height:20px;border-size:3px;border-color:#ffffff;border-bottom-style:solid;" width="70%">{{request.number_state}}</td>

```

Comment:
`pvt_number_state` is the old field that is now renamed to `pvt_state` (don't ask me why). Maybe we should read both?